javascript/javascript1/week3/removenames.js

```diff
@@ -0,0 +1,22 @@
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+  function removeName(names, nameToRemove) {
+    const index = names.indexOf(nameToRemove); 
+    if (index !== -1) { 
+      names.splice(index, 1); 
+    }
+  }
+  
+e
```

Comment:
Doesn't get more concise than this; good job! ⭐ There is an issue on this line though; I assume this `e` slipped past! 😄